Gemfile

```diff
@@ -2,6 +2,7 @@ source 'https://rubygems.org'
 
 gem 'opal', '0.9.0.beta2'
 #gem 'opal', :github => 'opal'
+gem 'rack', '1.6.4' # Opal compiler is *not* compatible with Rack 2.x
```

Comment:
@mojavelinux Do you know why suddenly `bundle install` is downloading rack 2.x ? Maybe a "non-strict" transitive dependency somewhere ?